ansible.cfg

```diff
@@ -0,0 +1,10 @@
+
+
+[defaults]
+inventory = aws_ec2.yml  # Path to your inventory file
+host_key_checking = False  # Optional: disable host key checking for SSH
+remote_user = ubuntu # Default user for SSH connections
```

Comment:
No need to degine the remote_user as `ubuntu`  as we are going to use the same role to redhat and centos as well. So default user name will change